package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "apify-client",
-	"version": "2.0.1",
+	"version": "2.1.0",
```

Comment:
no need for version bump as 2.0.0 is not yet out

Reply:
2.0.0 is out on NPM as far as I know.

Reply:
The beta is
![image](https://user-images.githubusercontent.com/17960496/132712402-d87400ca-f3b5-4779-9a63-008bd092cffc.png)

Reply:
It's not tagged as it was published by accident, but it's there 
<img width="894" alt="image" src="https://user-images.githubusercontent.com/23726914/132724383-47f56e2b-1295-41ad-b50d-7c924cf4da3e.png">